libass/ass.h

```diff
@@ -238,6 +238,14 @@ typedef enum {
      */
     ASS_FEATURE_BIDI_BRACKETS,
 
+    /**
+     * If no explicit ("\n") or implicit (" ") wrap point is found, wrap at the
+     * last seen character when doing smart wrap.
+     *
+     * Has no effect if wrap_mode is 2.
+     */
+    ASS_FEATURE_FORCE_WRAP_ON_OVERFLOW,
```

Comment:
I think `FORCE` is excess here, especially if we're planning to have a bunch of `WRAP` features.

Reply:
I agree that `WRAP` should come first, but to bikeshed: without `FORCE` anywhere the name no longer is a good description of the feature, as except for `\q2` we always try to wrap on overflow; this feature just forces wrap even if no legal breakpoints were found.
 But to keep the name short, maybe `ASS_FEATURE_WRAP_BREAK_WORDS` is better after all?